node/external/blockAPI/metaBlock_test.go

```diff
@@ -418,7 +418,26 @@ func TestMetaAPIBlockProcessor_GetBlockByRound_GetBlockByNonce_EpochStartBlock(t
 			},
 		},
 		EpochStart: block.EpochStart{
-			LastFinalizedHeaders: []block.EpochStartShardData{{}},
+			LastFinalizedHeaders: []block.EpochStartShardData{{
```

Comment:
Please do:
* add one more finalized header
* add one more miniblock header in each finalized header

Reply:
done.